core/src/main/scala/com/databricks/labs/remorph/generators/sql/ExpressionGenerator.scala

```diff
@@ -28,10 +28,40 @@ class ExpressionGenerator(val callMapper: ir.CallMapper = new ir.CallMapper())
       case a: ir.Alias => alias(ctx, a)
       case d: ir.Distinct => distinct(ctx, d)
       case s: ir.Star => star(ctx, s)
+      case col: ir.Column => column(ctx, col)
+      case da: ir.DeleteAction => deleteAction(ctx, da)
```

Comment:
```suggestion
      case da: ir.DeleteAction => "DELETE"
```

Reply:
OK - I did debate that myself, but the other options did not do it.